src/Microsoft.AspNet.Owin/OwinExtensions.cs

```diff
@@ -67,21 +68,32 @@ public static IApplicationBuilder UseOwin(this IApplicationBuilder builder, Acti
 
         public static IApplicationBuilder UseBuilder(this AddMiddleware app)
         {
+            return app.UseBuilder(configureServices: null);
+        }
+
+        public static IApplicationBuilder UseBuilder(this AddMiddleware app, Action<IServiceCollection> configureServices)
```

Comment:
Isn't that DI.Abstractions? https://github.com/aspnet/Options/blob/dev/src/Microsoft.Extensions.OptionsModel/project.json#L11